### PR TITLE
Upgrade for origami breaking cascade

### PR DESCRIPTION
**Pull request by edds**
The demos don't work but that was broken in a previous change on this
repo. 🐿 v2.12.5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
   },
   "dependencies": {
     "@financial-times/x-engine": "^1.0.0-beta.5",
-    "@financial-times/n-eventpromo": "^5.0.12",
+    "@financial-times/n-eventpromo": "^6.0.0-beta.1",
     "handlebars-loader": "^1.7.0",
     "js-cookie": "^2.2.0",
     "o-date": "git+https://github.com/Financial-Times/o-date.git#v1.2.2",
```
Comment (JakeChampion) [inline package.json]: o-date is at v4 now, do we need to update it here?

Comment (edds) [reply package.json]: Missed that, looks like it needs replacing with ft-date-format.

File: tests/lib/magnet-api-client.spec.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import fetchMock from 'fetch-mock';
+const fetchMock = require('fetch-mock');
```
Comment (JakeChampion) [inline tests/lib/magnet-api-client.spec.js]: How come this had to change to using require?

Comment (edds) [reply tests/lib/magnet-api-client.spec.js]: Pah, I left my revert of the ESM change in as that let me run the demos. Removed.

**Pull request merged**